WalletWasabi/WabiSabi/Client/CoinJoin/Client/CoinJoinCoinSelector.cs

```diff
@@ -80,7 +80,7 @@ public ImmutableList<TCoin> SelectCoinsForRound<TCoin>(IEnumerable<TCoin> coins,
 			.Where(x => x.IsRedCoin(SemiPrivateThreshold))
 			.ToArray();
 
-		if (stopWhenAllMixed && semiPrivateCoins.Length + redCoins.Length == 0)
+		if (semiPrivateCoins.Length + redCoins.Length == 0)
```

Comment:
This change is solving mixing below 5k stats. If there are no semiPrivateCoins or redCoins in the selection then CoinJoin will not continue.